### DISCUSSION TITLE
Keywords e Descriptions

### DISCUSSION THREAD

**Discussion by AblabiX** [cotes2020/jekyll-theme-chirpy#2086] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

Hi, is it possible to add descriptions and keywords for SEO points, on each page post in the Front Matter? Thank you in advice

### Operations you have already tried

nothing

**Comment by kungfux** [comment]
I guess there is no way to specify keywords.

The description is automatically created for each post and can be overwritten as follows:

```md
---
description: Custom description
---
```
